### DISCUSSION TITLE
DriverException ERROR after updating to aio 30

### DISCUSSION THREAD

**Discussion by lukasig** [nextcloud/all-in-one#5277] [Questions] [answered]
I am running nextcloud-AIO inside docker on ubuntu server 24.04
After updating to v 30.0.0 my logs keep showing this type of error (from application '**core**'):

> **DriverException**
> An exception occurred while executing a query: SQLSTATE[42803]: Grouping error: 7 ERROR: column "cr.id" must appear in the GROUP BY clause or be used in an aggregate function LINE 1: SELECT "cr".*, "co"."calendardata", "c"."displayname", "c"."... ^
> Error while running background job OCA\DAV\BackgroundJob\EventReminderJob (id: 7, arguments: null)

same question in [nc forums](https://help.nextcloud.com/t/error-after-upgrading-to-30-0-driver-exception/203947)

**Comment by szaimen** [comment, marked as answer]
This looks like a bug in server: https://github.com/nextcloud/server/issues/47275
